UPGRADE.md

```diff
@@ -14,6 +14,13 @@ We now support version `v0.6.0` of the Gateway API. See the [upstream API
 changes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.6.0) for
 more info.
 
+### Auth configuration of DP server in Kuma CP
+
+`dpServer.auth` configuration of Kuma CP was deprecated. You can still set config in this branch, but it will be removed in the future.
+It's recommended to migrate to `dpServer.authn` if you explicitly set any of the configuration in this config branch.
+* `dpServer.auth.type` is now split into two: `dpServer.authn.dpProxy.type` and `dpServer.authn.zoneProxy.type` and is still autoconfigured based on the environment.
+* `dpServer.auth.useTokenPath` is now `dpServer.authn.enableReloadableTokens`
```

Comment:
Using `branch` is not super easy to read. Is `section` or `subsection` a better word? Could you also show an example of before/after ?